server/lib/queries.js

```diff
@@ -1137,8 +1146,8 @@ const getTransactionsTimeSeries = async (kind, type, hostCollectiveId, timeUnit,
          AND type = :type
          AND "deletedAt" IS NULL
          ${collectiveIds ? `AND "CollectiveId" IN (:collectiveIds)` : ``}
-         ${dateFrom ? `AND "createdAt" >= :dateFrom` : ``}
-         ${dateTo ? `AND "createdAt" <= :dateTo` : ``}
+         ${dateFrom ? `AND "createdAt" >= :startDate` : ``}
+         ${dateTo ? `AND "createdAt" <= :endDate` : ``}
```

Comment:
This is confusing to have different names in the same block, dateFrom/startDate. This should be avoided.

Reply:
I've changed this to have the same names. 👍🏽